pkg/kev/config/extensions_test.go

```diff
@@ -0,0 +1,117 @@
+package config_test
+
+import (
+	"bytes"
+
+	"github.com/appvia/kev/pkg/kev/config"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"gopkg.in/yaml.v3"
+)
+
+var _ = Describe("Extentions", func() {
+
+	Describe("parsing", func() {
+		var (
+			parsedCfg  config.K8SServiceConfig
+			k8s        config.K8SServiceConfig
+			extensions = make(map[string]interface{})
+			err        error
+		)
+
+		BeforeEach(func() {
+			k8s = config.K8SServiceConfig{}
+		})
+
+		JustBeforeEach(func() {
+			var buf bytes.Buffer
+			err := yaml.NewEncoder(&buf).Encode(map[string]interface{}{
+				"x-k8s": k8s,
+			})
+			Expect(err).To(Succeed())
```

Comment:
Canonically we've been using `ToNot(HaveOccurred())` in our tests.

From the [Gomega docs](https://onsi.github.io/gomega/#handling-errors) `Succeed()` asserts a function has returned without an error and is ideally used as:
```go
Expect(DoSomethingHard()).Should(Equal("foo"))
```

So IMHO we should either stick with `ToNot(Have...` or use `Succeed()` as set in the Gomega docs - but ideally not mix the two.

Reply:
I blame them for having at least 8 different ways of expressing the same thing!